js/components/seismic-eruptions-map.js

```diff
@@ -39,14 +41,19 @@ export default class SeismicEruptionsMap extends PureComponent {
     super(props)
     this.state = {
       selectedEarthquake: null,
-      selectedVolcano: null
+      selectedVolcano: null,
+      scaleWidth: 0,
+      scaleHeight: 0,
+      zoom: 0,
+      mapViewport: { center: { lat: config.centerLat, lon: config.centerLng }, zoom: config.centeredInitialZoom }
```

Comment:
Do we need zoom included in the state twice?

Reply:
You're right, I don't think we need to be setting state. I'll adjust the PR accordingly, thanks!